### ISSUE TITLE
bash: ./show_wifi_passwd.sh: No such file or directory

### ISSUE THREAD

**Issue by ghost** [cvmiller/show_wifi_password#3]
I assume I need to download the .sh file, but I'm not sure where to put it. Could you point me in the right direction?

**Comment by cvmiller** [comment]
That's correct, you need to download the script, and place it anywhere you want on the Crouton side. ~/Downloads is fine. You will then run the script inside Crouton. Hope this helps.
